clcache.py

```diff
@@ -448,15 +451,18 @@ def save(self):
 class AnalysisResult:
     Ok, NoSourceFile, MultipleSourceFilesSimple, \
         MultipleSourceFilesComplex, CalledForLink, \
-        CalledWithPch, ExternalDebugInfo = range(7)
+        CalledWithPch, ExternalDebugInfo = list(range(7))
 
 
-def getFileHash(filePath, additionalData=None):
+def getFileHash(filePath, additionalDataString=None):
     hasher = HASH_ALGORITHM()
     with open(filePath, 'rb') as inFile:
         hasher.update(inFile.read())
-    if additionalData is not None:
-        hasher.update(additionalData)
+    if additionalDataString is not None:
```

Comment:
Right, in this case it is not helping as much as in the file case. I'll revert this renaming. But let me know what you think of the file case above.